### PR TITLE
Resolution For Issue Submission #138

### PR DESCRIPTION
**Pull request by M3TIOR**

### CODE REVIEW

**Comment by M3TIOR** [comment]
Branch has no conflicts but throws build errors on Travis. I should have been aware that you support legacy versions of NodeJS. The syntax I used is EMCAScript7 compliant. I don't believe the older version of node are compatible. My code may need to be rewritten and I'm too tired at the moment. I hope this helps! If you're looking for a new maintainer I'd be happy to help. The argparse python library is one of my favorites.

**Comment by M3TIOR** [comment]
Finally :sweat_smile: took me long enough to get back to this. :laughing:

**Comment by M3TIOR** [comment]
I don't want to run native NPM on my system because I have very low onboard memory and I'm trying to conserve every kilobyte I can get. So someone else will have to do the chore of exporting the **package-lock.json** unless of course you want my **pnpm-lock.yml** which will be of no use to regular NPM users :sweat_smile:.

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "eslint": "^2.13.1",
     "istanbul": "^0.4.5",
     "mocha": "^3.1.0",
-    "ndoc": "^5.0.1"
+    "ndoc": "^5.0.1",
+    "spawn-sync": "~1.0.0"
```
Comment (earonesty) [inline package.json:33]: move to devDependencies

**Comment by puzrin** [comment]
We will care about that in 2.0.0. ~ 1 week. You can see progress in temporary `rewrite` branch. This will be port of 3.9.0, very close, and i hope with all problems solved.

If you know any additional cases, specific for node.js and really missed (or no tested) in python - let us know here how to reproduce.

**Comment by rlidwka** [comment]
full rewrite done, bug has fixed itself

**Pull request closed without merging**